examples/websocket-provider/client.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+from time import sleep
+import websockets
+import y_py as Y
+import queue
+import concurrent.futures
+import threading
+
+# Code based on the [`websockets` patter documentation](https://websockets.readthedocs.io/en/stable/howto/patterns.html)
+
+class YDocWSClient:
+
+    def __init__(self, uri = "ws://localhost:8765"):
+        self.send_q = queue.Queue()
+        self.recv_q = queue.Queue()
+        self.uri = uri
+        def between_callback():
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
+
+            loop.run_until_complete(self.start_ws_client())
+            loop.close()
+        _thread = threading.Thread(target=between_callback)
+        _thread.start()
+
+        
+
+
+    def send_updates(self, txn_event: Y.AfterTransactionEvent):
+        update = txn_event.get_update()
+        if update != b'\x00\x00':
+            self.send_q.put_nowait(update)
```

Comment:
@Horusiath I noticed that the `observe_after_transaction` is running in a tight loop with an empty update message when nothing has changed. Is this happening due read-only transaction operations in the `draw()` loop that don't change the state?